modules/kn-service-offline-about.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="kn-service-offline-about_{context}"]
+= About offline services
+
+The `kn` CLI enables you to work with services in the offline mode, also called the gitops mode. In this mode, no operations with Knative services happen on the cluster. Instead, descriptor files of the respective services are locally created or modified. These resource files can then be used with commands such as `kn service create -f` and `kn service apply -f` to work with the actual services on the cluster.
```

Comment:
In addition to `kn` commands, it can be also used with plain `oc apply -f`. I'd mention it as an viable alternative, e.g. for CI pipelines that might not have `kn` installed etc. 
The use case is: 
1. generate with `kn`
2. store in SVC
3. `oc apply -f` when needed.